.github/workflows/unit-test.yml

```diff
@@ -84,6 +89,11 @@ on:
         required: false
         type: string
         default: ''
+      pytest_benchmark_flags:
+        description: Benchmark arguments to pass to PyTest
+        required: false
+        type: string
+        default: --benchmark-enable --benchmark-only --benchmark-json=benchmarks.json
```

Comment:
Instead of adding this variable, would it be possible to use the existing `pytest_additional_args`? We can sync offline on this if you'd like

Reply:
I like to have variables with meaningful names, and I'm afraid that if we want to use `pytest_additional_args` in the future we might have problems. Let me know your thoughts.

Reply:
do you expect these to change? why not just stick the options right in the command?